src/test/java/duke/StorageTest.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.FileNotFoundException;
+
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StorageTest {
+    @Test
+    public void textToTask_test1() throws DukeException {
```

Comment:
Perhaps your method name for your test could be what you're trying to test. It could be something like `testNormalTodo` or `testInvalidTodo`.